### ISSUE TITLE
Reset Filters action doesn't clear the selections with Nova 3.2.1

### ISSUE THREAD

**Issue by andrealandonio** [rcknr/nova-multiselect-filter#19]
Hi, same problem seen in a previous issue... my Nova version is 3.2.1 with Laravel 7. I'm using the same code snippet explained in the Readme but resetting filters don't clean the selections.. like in https://github.com/rcknr/nova-multiselect-filter/issues/10

Any suggestions?

**Comment by skauk** [comment]
@andrealandonio Is any error shown in the console?

**Comment by andrealandonio** [comment]
No errors unfortunately...

**Comment by andrealandonio** [comment]
The strange thing is that when I enter the page the first time (also after a filters reset) I see the filters icon enabled but no items are selected.

![Screenshot from 2020-04-19 16-20-42](https://user-images.githubusercontent.com/9296058/79690305-04f42c80-825a-11ea-812f-ed15b8ac9895.png)

![Screenshot from 2020-04-19 16-20-57](https://user-images.githubusercontent.com/9296058/79690308-0a517700-825a-11ea-808e-2df9aced3cd9.png)

And, in the URL there is a parameter "<Resource>_filter" with an hash value..
like:
http://127.0.0.1:8000/admin/resources/credits?credits_page=1&credits_filter=W3siY2xhc3MiOiJBcHBcXE5vdmFcXEZpbHRlcnNcXENyZWRpdFR5cGUiLCJ2YWx1ZSI6W119XQ%3D%3D

Have you even seen a similar issue?

**Comment by skauk** [comment]
Yes. Happens sometimes when debugging filters. Remove all URL parameters to
reset.

Andrea Landonio <notifications@github.com> schrieb am So., 19. Apr. 2020,
16:24:

**Comment by andrealandonio** [comment]
Hi @rcknr .... what do you mean with remove all URL parameters to reset? removing parameters is a good way for resetting filers but the very strange behavior is that if I reload  the page the parameters come back... :-(

**Comment by bastinald** [comment]
This is still happening. Removing the URL parameters does not correct the issue as stated above.

Is this project still maintained?
